### ISSUE TITLE
FileResult

### ISSUE THREAD

**Issue by curlysemi** [curlysemi/R4Mvc5#7]
There's some "`contentType` cannot be null" error right now in the current R4Mvc5 for actions that say the return type is `FileResult`.

(Need to refer to a T4MVC generated FileResult to fix)

As a work-around, just change the returned type in the function signature to `ActionResult`.

**Comment by curlysemi** [comment]
This is what T4MVC generated:
```
internal partial class T4MVC_System_Web_Mvc_FileResult : System.Web.Mvc.FileResult, IT4MVCActionResult
{
    public T4MVC_System_Web_Mvc_FileResult(string area, string controller, string action, string protocol = null): base(" ")
    {
        this.InitMVCT4Result(area, controller, action, protocol);
    }
     
    protected override void WriteFile(System.Web.HttpResponseBase response) { }
    
    public string Controller { get; set; }
    public string Action { get; set; }
    public string Protocol { get; set; }
    public RouteValueDictionary RouteValueDictionary { get; set; }
}
```

We need to change that `null` it to a `" "`.

**Comment by curlysemi** [comment]
Fixed in 4f48a7aa6cc48b9da6e9fa9327f0ee36a977945f.
